libs/@guardian/identity-auth/src/identityAuth.ts

```diff
@@ -94,23 +94,31 @@ export class IdentityAuth<
 	 */
```

Comment:
You may want to update the docblock here - on line 87 it reads `If the user has a GU_SO cookie, they have recently signed out, so we should clear their tokens (side effect)`, which is not exactly true any more.

Reply:
very good spot!

will update the comment